modules/react/breadcrumbs/lib/BreadcrumbsItem.tsx

```diff
@@ -39,19 +66,13 @@ export const BreadcrumbsItem = createSubcomponent('li')({
   },
 })<BreadcrumbsItemProps>(({children, ...elemProps}, Element) => {
   return (
-    <Flex as={Element} alignItems="center" whiteSpace="nowrap" {...elemProps}>
+    <Element {...mergeStyles(elemProps, breadcrumbsItemStencil())}>
       {children}
       <SystemIcon
         icon={chevronRightSmallIcon}
-        color={colors.licorice200}
-        colorHover={colors.licorice200}
-        size={20}
-        height={space.l}
-        width={space.l}
-        shouldMirror={useIsRTL()}
```

Comment:
how would this be translated in a stencil?

Reply:
I would imagine that the things that are CSS properties you would put into a stencil, otherwise, I would imagine you leave it?